### PR TITLE
[Apps] Disable data-apps prior to release

### PR DESCRIPTION
**Pull request by snoe**
This ensures that the api endpoints and the relevant models cannot be created.

We enable during testing, so that we can ensure the integration points like permissions, search, collection items, etc... will continue to be tested.

### CODE REVIEW

File: src/metabase/models/action.clj [hunk 1]
```diff
@@ -73,6 +87,7 @@
 (defn insert!
   "Inserts an Action and related HTTPAction or QueryAction. Returns the action id."
   [action-data]
+  (check-data-apps-enabled)
```
Comment (metamben) [inline src/metabase/models/action.clj:90]: Wouldn't pre-insert throw anyway?

**Review by metamben — APPROVED** [review]
👍

File: src/metabase/models/action.clj (hunk 1 above)
Comment (snoe) [reply src/metabase/models/action.clj:90]: Ya, I figured it was a bit more clear to have it here, since the Action models are a bit unidiomatic.

**Comment by metamben** [comment]
One more thing: should this PR shortcut the `/permission/execution/graph` endpoints too?

**Pull request merged**